### ISSUE TITLE
Easy installation as Zsh plugin

### ISSUE THREAD

**Issue by psprint** [so-fancy/diff-so-fancy#287]
Hello,
it was hurting me that I couldn't install `diff-so-fancy` in user's home, via my Zsh plugin manager – [Zplugin](https://github.com/zdharma/zplugin). Typically, when I install some artifact that IS NOT a Zsh-plugin, I construct invocation that runs make, configure, executes something on install and update. I can install Vim or Zsh itself this way, etc.

So `diff-so-fancy` didn't have Makefile (but turns out it isn't needed) and required update of `.gitconfig`. To solve this I've created [zsh-diff-so-fancy](https://github.com/zdharma/zsh-diff-so-fancy) Zsh-plugin that has `diff-so-fancy` as git submodule and provides subcommand `dsf` for git. 

So basically one can install `diff-so-fancy` with single command, update it the same way (`zplugin update zdharma/zsh-diff-so-fancy`), no system-wide package manager is needed, and subcommand `dsf` is added to git with the same single command. I think this is interesting so I write this issue. The point is that two lines in dotfile (`.zshrc`) substitute system-wide package manager and provide git subcommand. This is like Puppet or Chef, declarative approach, one can copy the dotfile to other machine and have `diff-so-fancy` up and running in few minutes. Maybe this will inspire you to provide some Makefile or git subcommand.

**Comment by scottchiefbaker** [comment]
I would definitely accept a pull-request to add this sort of thing to our [Pro Tips](https://github.com/so-fancy/diff-so-fancy/blob/master/pro-tips.md) documentation.

**Comment by psprint** [comment]
Ok, I've created the pull request, so I'll close this issue.
